src/components/__snapshots__/OcModal.spec.js.snap

```diff
@@ -1,9 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`OcModal displays input 1`] = `
-<div aria-labelledby="oc-modal-title" class="oc-modal-background">
```

Comment:
Not sure how this change came here - is the snapshot broken on the `a11y-updates` branch?

Reply:
Nevermind, fixed the outdated snapshot and pushed to `a11y-updates` (= target branch)